Qase_Test/Qase_Test/SRC/Resources/appSettings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "Url": "https://app.qase.io/login",
+    "HomeUrl": "https://app.qase.io/projects",
+    "Browser": "chrome",
+    "Timeout": "10",
+    "Email": "balexer@mail.ru",
+    "Password": "balexerPassword123",
+    "WrongPassword": "wrongPass",
+    "ProjectName": "TestProject",
+    "ProjectCode": "TS",
+    "AnotherProjectCode": "MEMBER",
+    "WrongProjectName": "A",
+    "ProjectDescription": "Description",
+    "TestCaseName": "Test Case",
+    "EmptyTestCaseName": "",
+    "ExistingProjectCode": "AA",
+    "Token": "34b9c83661ef70c9d77a0be959aa87781525be9a",
+    "MemberToken": "900bb142759956dc1bf543c10a84c14063cb0361"
+}
```

Comment:
Make a beautiful JSON structure. It is not clear what is related to what.
Create user objects that have some of their own characteristics.
Create a project object - with your own. Empty line